Godeps/_workspace/src/github.com/miekg/dns/types.go

```diff
@@ -1567,9 +1567,10 @@ type CAA struct {
 
 func (rr *CAA) Header() *RR_Header { return &rr.Hdr }
 func (rr *CAA) copy() RR           { return &CAA{*rr.Hdr.copyHeader(), rr.Flag, rr.Tag, rr.Value} }
-func (rr *CAA) len() int           { return rr.Hdr.len() + 1 + len(rr.Tag) + len(rr.Value)/2 }
-func (rr *CAA) String() string     { return rr.Hdr.String() + strconv.Itoa(int(rr.Flag)) + " " + rr.Tag + " " + sprintCAAValue(rr.Value) }
```

Comment:
Whoops I thought I'd added a comment about this, this a patch for a bug in a patch I submitted upstream to `miekg/dns` a while ago (...). I've been dragging my feet on submitting it because I haven't written test cases yet... but I will! Soon! Once it's pulled in upstream we can just update via Godep instead of patching it directly here.